client/components/molecules/ProgressBar/index.tsx

```diff
@@ -41,12 +43,22 @@ interface ProgressBarProps {
 }
 
 const ProgressBar = ({ progress, totalPlaytime }: ProgressBarProps) => {
+    const dispatch = useDispatch();
+    const { nowPlaying, playTime, upNextTracks } = useSelector(state => state.musicPlayer);
+    
+    const onClickHandler = (e, newValue) => {
+        dispatch(setTrackPlaytime(newValue));
+    }
+
     return (
         <>
             <PlayerSlider
+                value={ playTime }
+                onChangeCommitted = {onClickHandler}
                 valueLabelDisplay="auto"
                 aria-label="player slider"
                 defaultValue={progress ? progress : 0}
+                max={totalPlaytime}
                 scale={(x) => convertToHHSS(Math.floor((totalPlaytime * x) / 100))}
```

Comment:
- 특이사항 1 
프로그래스 바의 스케일 속성에서 옵니다 !  `scale={(x) => convertToHHSS(Math.floor((totalPlaytime * x) / 100))}` 여기서 x가 value 값이 되는데 원래 value 값이 진행률일 때 계산 식을 저렇게 설정해놨는데 지금은 value가 progress가 아닌 playTime으로 되어 있어서 의도하던 값이 안나오는 것 같네용. 
```suggestion
                scale={(x) => convertToHHSS(Math.floor((x/totalPlaytime)*100))}
```
이렇게 고쳐주심 될거 같아요 !

Reply:
이부분 수정했습니다! 그냥 현재 재생시간 넣어주면 되더라구요!